projects/storefrontlib/src/shared/components/form/form-errors/form-errors.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { GlobalMessageService, GlobalMessageType } from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FormErrorsService {
+  constructor(protected globalMessageService: GlobalMessageService) {}
+
+  showFormErrorGlobalMessage(): void {
```

Comment:
I'd keep the method short. We're already in the contexts of FormErrors, so no need to repeat that in the method name. 
I also like the method to be less opinionated about target (global message). If a customer want to extend this method, as he likes to render it in a form message box, it's strange to extend a method which includes the target. 

So perhaps just: `FormErrorsService.show()`

Reply:
fixed, I changed it to `notifyUser()`